server/portal/apps/workspace/api/views.py

```diff
@@ -297,6 +297,11 @@ def post(self, request, *args, **kwargs):
                     homeDir = settings.PORTAL_DATAFILES_DEFAULT_STORAGE_SYSTEM['homeDir'].format(tasdir=tasdir, username=username)
                     job_post['archiveSystemDir'] = f'{homeDir}/tapis-jobs-archive/${{JobCreateDate}}/${{JobName}}-${{JobUUID}}'
 
+            execSystemId = job_post.get("execSystemId")
```

Comment:
how is this relevant to compress PR change? I'm missing the context.

Reply:
I remember having this in my other mutation hooks up to this point. I was running into problems with `execSystemId`, and I'm pretty sure this was the fix for that.

Reply:
You can revert this file, see my comment on getCompressParams.ts